Header header
# string class_name
int32 x1
int32 y1
int32 x2
int32 y2
int32 x3
int32 y3
float32 conf
